### ISSUE TITLE
Borders drawn incorrectly with pancurses backend

### ISSUE THREAD

**Issue by gyscos** [gyscos/Cursive#72]
![pancurses_dialog](https://cloud.githubusercontent.com/assets/564903/19283099/90f7cc1a-8fa5-11e6-832d-b882fa49311d.png)

**Comment by nabijaczleweli** [comment]
Probably because it doesn't use the `wide` feature. Try adding in `features = ["wide"]` in `[dependencies.pancures]`

**Comment by gyscos** [comment]
pancurses doesn't forward such an option, and I'm not sure how to specify the features for a sub-dependency.

**Comment by nabijaczleweli** [comment]
But features have one global namespace, IIRC

**Comment by gyscos** [comment]
```
% cargo run --example dialog --no-default-features --features pancurses
error: Package `pancurses v0.4.1` does not have these features: `wide`
```

Seems not :-/

**Comment by nabijaczleweli** [comment]
Open an issue at https://github.com/ihalila/pancurses, then :v

**Comment by gyscos** [comment]
Blocked on https://github.com/ihalila/pancurses/issues/6.

**Comment by gyscos** [comment]
Not sure the wide feature is enough, though: I tried with this diff, without any improvement:
```diff
diff --git a/Cargo.toml b/Cargo.toml
index 89c7e8c..969c8db 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,10 @@ features = ["wide"]
 optional = true
 
 [dependencies.pancurses]
-version = "0.4.0"
+# version = "0.4.0"
+git = "https://github.com/gyscos/pancurses"
+branch="patch-1"
+features = ["wide"]
 optional = true
 
 [dependencies.termion]
```

**Comment by gyscos** [comment]
This should be fixed in pancurses 0.6.0. Just waiting for it to be published on crates.io.
